pkg/scraper/jobcenter_scraper.go

```diff
@@ -94,12 +116,35 @@ func getJobcenterJobId(s string) (string, error) {
 	matches := r.FindStringSubmatch(s)
 
 	if len(matches) < 2 {
-		return "", errors.New(fmt.Sprintf("job id is empty: %s", s))
+		return "", fmt.Errorf("job id is empty: %s", s)
 	}
 
 	if matches[1] == "" {
-		return "", errors.New(fmt.Sprintf("no job id found: %s", s))
+		return "", fmt.Errorf("no job id found: %s", s)
 	}
 
 	return matches[1], nil
 }
+
+func scrapeJobcenterLastPageNumber() (int, error) {
+
+	urlString := fmt.Sprintf("%s/web/guest/search-job?q=&delta=%d&start=%d", jobcenterUrl, pageSize, 1)
+
+	doc, err := getDocument(urlString)
+
+	if err != nil {
+		return 0, errors.New("failed to get document")
```

Comment:
Should return original err. If you want to attach additional context, use `errors.Wrapf` from `github.com/pkg/errors`.

```go
return 0, errors.Wrapf(err, "failed to get document. url: %s", urlString)
```